contributing

```diff
@@ -0,0 +1,33 @@
+# Contributing to Project Open Data
+
+*By contributing to this project, you dedicate your work to the public domain and relinquish any copyright claims under the terms of the [CC0 Public Domain Dedication](http://creativecommons.org/publicdomain/zero/1.0/). See the [license file](LICENSE) for additional information.*
+
+## How to Contribute
+
+This project constitutes a collaborative work ("open source"). Federal employees and members of the public are encouraged to improve the project by contributing. For more information, please see [the FAQ](https://project-open-data.cio.gov/faq/)
+
+Contributions can be made, primarily in two ways:
+
+### Easy
+
+1. Browse the content at https://project-open-data.cio.gov/.
```

Comment:
Should we change this URL? We could temporarily use this GitHub repo until it lives elsewhere